tools/phenomenal/nmr/nmrml2batman.xml

```diff
@@ -6,14 +6,20 @@
        <container type="docker">container-registry.phenomenal-h2020.eu/phnmnl/nmrml2batman</container>
      </requirements>
      <command><![CDATA[   
-     export indir=`dirname $nmrMLzip`;
-     mkdir -p \$indir/tmp/;
-     unzip -q -d \$indir/tmp/ $nmrMLzip;
-     nmrML2batman.R -i \$indir/tmp/
+         galaxy_data_dir="\$(dirname "${nmrMLzip}")";
```

Comment:
hehehe you beat me to commit this @ilveroluca .... I have in my working copy a less convoluted proposition:

```
     mkdir -p \$PWD/tmp/;
     unzip -q -d \$PWD/tmp/ $nmrMLzip;
     nmrML2batman.R -i \$PWD/tmp/
```

let me know to commit on top (or not)

Reply:
I would rather have this tool receive a collection of files instead of a zip file though...
what do you think?

Reply:
Re "less convoluted":  Ok...I guess  `$PWD` should be guaranteed to be the galaxy data directory.  However, I'd keep the `&&` to make sure any errors make their way to the final step.

Re "receiving a collection of files":  maybe let's leave it as it is for this release.  We'd have to test, update test cases and get some feedback from the users. In the future we can propose this change to the others and see what they think about it.  It would make the tools more modular.

Reply:
Yes `\$PWD` will be the Galaxy working directory, I use this in other wrappers and it is like this consistently. I will add the proposed version then (I have tested it in minikube).